internal/datastore/common/sql_test.go

```diff
@@ -18,297 +18,307 @@ import (
 
 var toCursor = options.ToCursor
 
+type expected struct {
+	sql        string
+	args       []any
+	staticCols []string
+}
+
 func TestSchemaQueryFilterer(t *testing.T) {
```

Comment:
Can you add some tests for rel integrity bits?

Reply:
I did

Reply:
running `sql_test.go` show no coverage of the SQL integrity code